gcsfs/core.py

```diff
@@ -586,9 +788,9 @@ def rm(self, path, recursive=False):
             for p in self.walk(path):
                 self.rm(p)
```

Comment:
I wonder if you happen to know if there is a bulk-delete option in GCS?

Reply:
It would be very nice if this were the case.  Some of @jhamman 's benchmarks with Zarr spend a lot of time removing tiny files.

Reply:
Unfortunately the GCS api doesn't have a bulk-delete operation. There are a number of possibilities to speed up object deletion. The easiest, and what's implemented in gsutil, would be to issue a number of concurrent delete requests. You probably have a better grounding with if/how this should be integrated into existing async runloop, but [requests-futures](https://github.com/ross/requests-futures) would be an easy, standalone solution.